### PR TITLE
Update New-AzContainerGroup Example6

### PR DESCRIPTION
**Pull request by CaptainFanZzz**
## Description

Update New-AzContainerGroup Example6
Generation branch #17552 
Fix for https://github.com/Azure/azure-powershell/issues/17123
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/ContainerInstance/help/New-AzContainerGroupVolumeObject.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ New-AzContainerGroupVolumeObject -Name <String> [-AzureFileReadOnly] [-AzureFile
 ```
 
 ## DESCRIPTION
-Create a in-memory object for Volume
+Create a in-memory object for Volume. Currently, we support mounting Azure File share as a volume or specifying git repository as volume directory. The empty directory volume and secret volume are not supported yet.
```
Comment (BethanyZhou) [inline src/ContainerInstance/help/New-AzContainerGroupVolumeObject.md:23]: I'm hesitating to do this job. The description is auto-generated, I'm afraid that this change will be overridden in next change. Is there any way to customize description of cmdlets? @dolauli

Comment (dolauli) [reply src/ContainerInstance/help/New-AzContainerGroupVolumeObject.md:23]: The cmdlet is a custom cmdlet. So we could customize the description in https://github.com/CaptainFanZzz/azure-powershell/blob/update-New-AzContainerGroup/src/ContainerInstance/custom/New-AzContainerGroupVolumeObject.ps1#L19

Comment (BethanyZhou) [reply src/ContainerInstance/help/New-AzContainerGroupVolumeObject.md:23]: How about New-AzContainerGroup? It's a customized cmdlet after generated . Can we use the same way?

Comment (dolauli) [reply src/ContainerInstance/help/New-AzContainerGroupVolumeObject.md:23]: > 

Yes.

Comment (BethanyZhou) [reply src/ContainerInstance/help/New-AzContainerGroupVolumeObject.md:23]: Thanks @dolauli .

**Comment by BethanyZhou** [comment]
As @jaltin mentioned, could you create a separate PR for this issue? @CaptainFanZzz

**Comment by CaptainFanZzz** [comment]
> As @jaltin mentioned, could you create a separate PR for this issue? @CaptainFanZzz

Okay

File: src/ContainerInstance/help/New-AzContainerGroup.md [hunk 2]
```diff
@@ -122,7 +123,7 @@ Location Name    Zone ResourceGroupName
 eastus   test-cg      test-rg
 ```
 
-This commands creates a container group with a container instance, whose image is a custom image from a custom container image registry.
+This commands creates a container group with a container instance, whose image is a custom image from a custom container image registry. For the volume property, AzureFile and GitRepo are recommended. The EmptyDir property isn't supported.
```
Comment (BethanyZhou) [inline src/ContainerInstance/help/New-AzContainerGroup.md:126]:
```suggestion
This commands creates a container group with a container instance, whose image is a custom image from a custom container image registry.
```
Remove unrelated description for this example.

**Review by BethanyZhou — CHANGES_REQUESTED** [review]

**Review by BethanyZhou — APPROVED** [review]
lgtm

**Pull request merged**